src/JuMP.jl

```diff
@@ -38,13 +38,13 @@ Base.@deprecate(linearterms,       JuMP.linear_terms)
 
 include("utils.jl")
 
-const MOIVAR = MOI.VariableIndex
-const MOICON{F,S} = MOI.ConstraintIndex{F,S}
-const MOILB = MOICON{MOI.SingleVariable,MOI.GreaterThan{Float64}}
-const MOIUB = MOICON{MOI.SingleVariable,MOI.LessThan{Float64}}
-const MOIFIX = MOICON{MOI.SingleVariable,MOI.EqualTo{Float64}}
-const MOIINT = MOICON{MOI.SingleVariable,MOI.Integer}
-const MOIBIN = MOICON{MOI.SingleVariable,MOI.ZeroOne}
+const _MOIVAR = MOI.VariableIndex
+const _MOICON{F,S} = MOI.ConstraintIndex{F,S}
+const _MOILB = _MOICON{MOI.SingleVariable,MOI.GreaterThan{Float64}}
+const _MOIUB = _MOICON{MOI.SingleVariable,MOI.LessThan{Float64}}
+const _MOIFIX = _MOICON{MOI.SingleVariable,MOI.EqualTo{Float64}}
+const _MOIINT = _MOICON{MOI.SingleVariable,MOI.Integer}
+const _MOIBIN = _MOICON{MOI.SingleVariable,MOI.ZeroOne}
 
 MOIU.@model(JuMPMOIModel,
```

Comment:
Unexport `JuMPMOIModel` also?

Reply:
I am unsure since it might be useful if someone want to use a CachingOptimizer in direct mode. However, why would someone use direct mode if a CachingOptimizer is used anyway...
If we unexport, it should be `JuMP._MOIModel` since `JuMP._JuMPMOIModel` is redundant.